internal/runners/history/history_test.go

```diff
@@ -22,7 +21,7 @@ func Test_printCommits(t *testing.T) {
 		name        string
 		args        args
 		wantStrings []string
-		wantFailure *failures.Failure
+		wantFailure error
```

Comment:
nit: maybe we can get rid of the `wantFailure` argument, as it is unused in this test...

Reply:
`wantFailure` is used, if it is nil in a test that we want no errors in we continue with the output checks